src/ResourceManager/Compute.ManagedService/Commands.Compute.ManagedService/help/ConvertTo-AzureRmVhd.md

```diff
@@ -0,0 +1,121 @@
+---
+external help file: 
+Module Name: 
+online version: 
+schema: 2.0.0
+---
+
+# ConvertTo-AzureRmVhd
+
+## SYNOPSIS
+Convert Hyper-V VM to Azure supported virtual hard disk files
+
+## SYNTAX
+
+```
+ConvertTo-AzureRmVhd -HyperVVMName <String> -ExportPath <String> [-HyperVServer <String>] [-Force] [-AsJob]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+Convert Hyper-V VM to Azure supported virtual hard disk files
+
+## EXAMPLES
+
+### Example 1
+```
+PS C:\> ConvertTo-AzureRmVhd -HyperVVMName 'test' -ExportPath '.';
+.\test\Virtual Hard Disks\Converted\os.vhd
+.\test\Virtual Hard Disks\Converted\disk.vhd
+```
+
+Convert Hyper-V VM named 'test' to Azure supported virtual hard disk files to the current folder.
+
+## PARAMETERS
+
+### -AsJob
+Run cmdlet in the background and return a Job to track progress.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases: 
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ExportPath
+The export folder path that will contain the disk files.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases: 
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Force
+Force the export even if existing folder is found.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases: 
+
+Required: False
+Position: Named
+Default value: False
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -HyperVServer
+The Hyper-V server DNS name, with 'localhost' as the default value.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases: 
+
+Required: False
+Position: Named
+Default value: Localhost
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -HyperVVMName
+The Hyper-V name name.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases: 
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+
+## INPUTS
+
+## OUTPUTS
+
```

Comment:
Outpiuts are still needed

Reply:
OK. Are these the last set of issues that we need to take care of? We may be running out of time for the upcoming release (again)...